jenkins_jobs/upload_build_artifacts/script.sh

```diff
@@ -1,32 +1,35 @@
-# Create status json
+# ISO 8601 date with  minute precision
+CURRENT_DATE=$(date --utc -Iminutes)
```

Comment:
The upload job is not restricted to run concurrently and with 4 slaves we may (unlikely) have up to 4 build jobs finishing at the same time. This will get worse once we have multiple builds running in the same slave. I think minute precision is not enough, we'd need second or, to be absolutely safe, the full timestamp.